commit 62c7c0a8709b254060cb5f76e9ed6a39f6aae20f
Author: Carlos Maiolino <cmaiolino@redhat.com>
Date:   Mon Feb 4 08:54:18 2019 -0800

    xfs: Fix xqmstats offsets in /proc/fs/xfs/xqmstat
    
    commit 41657e5507b13e963be906d5d874f4f02374fd5c upstream.
    
    The addition of FIBT, RMAP and REFCOUNT changed the offsets into
    __xfssats structure.
    
    This caused xqmstat_proc_show() to display garbage data via
    /proc/fs/xfs/xqmstat, once it relies on the offsets marked via macros.
    
    Fix it.
    
    Fixes: 00f4e4f9 xfs: add rmap btree stats infrastructure
    Fixes: aafc3c24 xfs: support the XFS_BTNUM_FINOBT free inode btree type
    Fixes: 46eeb521 xfs: introduce refcount btree definitions
    Signed-off-by: Carlos Maiolino <cmaiolino@redhat.com>
    Reviewed-by: Eric Sandeen <sandeen@redhat.com>
    Signed-off-by: Dave Chinner <david@fromorbit.com>
    Signed-off-by: Luis Chamberlain <mcgrof@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/fs/xfs/xfs_stats.c b/fs/xfs/xfs_stats.c
index 4e4423153071..740ac9674848 100644
--- a/fs/xfs/xfs_stats.c
+++ b/fs/xfs/xfs_stats.c
@@ -119,7 +119,7 @@ static int xqmstat_proc_show(struct seq_file *m, void *v)
 	int j;
 
 	seq_printf(m, "qm");
-	for (j = XFSSTAT_END_IBT_V2; j < XFSSTAT_END_XQMSTAT; j++)
+	for (j = XFSSTAT_END_REFCOUNT; j < XFSSTAT_END_XQMSTAT; j++)
 		seq_printf(m, " %u", counter_val(xfsstats.xs_stats, j));
 	seq_putc(m, '\n');
 	return 0;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1239_62c7c0a8709b254060cb5f76e9ed6a39f6aae20f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1239_62c7c0a8709b254060cb5f76e9ed6a39f6aae20f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
